vcx/libvcx/src/messages/update_profile.rs

```diff
@@ -80,6 +80,17 @@ impl UpdateProfileData{
         }
     }
 
+    pub fn use_public_did(&mut self, did: &Option<String>) -> &mut Self {
```

Comment:
This would be more 'rust' like:
```
    pub fn use_public_did(&mut self, did: &Option<String>) -> &mut Self {
        if let Some(x) = did {
            self.payload.configs.push(AttrValue {
                name: "publicDid".to_string(),
                value: x.to_string(),
            });
        };
        self
    }
```